docs/01_setup.md

```diff
@@ -148,6 +153,29 @@ https://vercel.com/new
   <img width="45%" src="./images/vercel_configure.png" />
 </div>
 
+## Poll finalization
+
+Once the voting time has ended, as a coordinator, first you need to merge signups and messages (votes). Head to MACI contracts repository and run the merge command with the deployed poll:
+
+```bash
+cd contracts  && \
+pnpm merge:[network] --poll [poll-id]
+```
+
+> [!IMPORTANT]
+> If you use the same MACI contract, each time you deploy a poll the id increases by one.
+
+Then the coordinator generates proofs for the message processing, and tally calculations. This allows to publish the poll results on-chain and then everyone can verify the results when the poll is over:
+
+```bash
+pnpm run prove:[network] --poll [poll-id] \
+    --coordinator-private-key [coordinator-maci-private-key] \
+    --tally-file ../cli/tally.json \
+    --output-dir ../cli/proofs/ \
```

Comment:
I think the wasm and zkey files are still needed?

Reply:
yes! I made an update for the tasks so it can grab it directly from the config file, will make a PR after I finish testing